docs/spec/BeakerSpec.tex

```diff
@@ -182,29 +195,24 @@ \subsection{Procedure Table}\label{procedure-table}
     Storage Location & Description\tabularnewline
     \hline
     \hline
-    \texttt{0x005bac198c377b7d}     & length/number of procedures
+    \texttt{0x00 00 00 00 01}     & length/number of procedures
```

Comment:
`0x00 00 00 00 00` is already specified for # of Procedures in Line :108

Reply:
But only using KeyIndex, which starts at 1, not 0. Here we could just say from 0 upwards is the procedure list, but I thought it better to be explicit, it means given any key you can look it up on this table and see exactly what it means.

The procedure list is formatted like a Solidity array (or indeed like most non-C style vectors in programming languages), which a length value followed by the values.